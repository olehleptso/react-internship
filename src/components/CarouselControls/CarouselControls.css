.carousel-controls {
    height: 15vh;
    width: 100%;
}

.controls-inner {
    height: 100%;
    display: flex;
    justify-content: space-between;
    margin: auto 6%;
}

.controls-inner img {
    width: 32px;
    height: 32px;
    margin: auto;
    cursor: pointer;
}

.btn {
    border: none;
    background-color: #fff;
    cursor: pointer;
    font-size: 16px;
    font-weight: bold;
    color: #191919;
    font-family: 'Montserrat', sans-serif;
    text-transform: uppercase;
    letter-spacing: 1px;
    padding: 0;
    margin: auto 0;
}
