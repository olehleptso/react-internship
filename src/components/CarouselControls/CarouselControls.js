import './CarouselControls.css';

function CarouselControls ({prev, next}) {
    return (
            <div className='carousel-controls'>
                <div className='controls-inner'>
                    <button className='previous btn' onClick={prev}>{'< '}previous project</button>
                    <button className='next btn' onClick={next}>next project{' >'}</button>
                </div>
            </div>
    )
}

export default CarouselControls;
