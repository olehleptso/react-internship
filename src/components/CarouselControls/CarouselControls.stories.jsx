import React from 'react';
import CarouselControls from './CarouselControls';

export default {
  title: 'CarouselControls',
  component: CarouselControls,
};

const Template = (args) => <CarouselControls {...args} />;

export const Standart = Template.bind({});
