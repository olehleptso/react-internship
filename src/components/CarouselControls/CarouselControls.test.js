import { render, screen } from '@testing-library/react';
import CarouselControls from './CarouselControls';

test('renders CarouselControls component', () => {
    render(<CarouselControls />);
    const CarouselControlsSec = screen.getByTestId('CarouselControls');
    expect(CarouselControlsSec).toBeInTheDocument();
});