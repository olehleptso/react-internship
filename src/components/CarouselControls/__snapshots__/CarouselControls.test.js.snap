// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders search CarouselControls component 1`] = `
<DocumentFragment>
  <div
    class="carousel-controls"
    data-testid="CarouselControls"
  >
    <div
      class="controls-inner"
    >
      <button
        class="previous btn"
      >
        &lt; previous project
      </button>
      <img
        src="squares.png"
      />
      <button
        class="next btn"
      >
        next project &gt;
      </button>
    </div>
  </div>
</DocumentFragment>
`;
